src/main/webapp/dev/js/common/onStart.js

```diff
@@ -61,4 +65,12 @@ $(document).on('ajaxComplete ready', () => {
             $(this).addClass('tool-tip-decorate');
         }
     });
+
+    /**
```

Comment:
Try not to use JSDoc-style comments if it's not for header comments; stay at `/* ... */` or `// ...`.